build.sbt

```diff
@@ -1,4 +1,4 @@
-val awsSdkVersion = "1.11.486"
+val awsSdkVersion = "1.11.618"
```

Comment:
should we use ivy syntax to specify 1.11.* so it automatically gets bumped?

Reply:
I think we should do that in libraries, but I'd prefer not do it in starport where we ship a binary runnable, it's better for making static and deterministic versions.

Reply:
Gotcha. Think we debated this before, makes perfect sense.